charts/jx-app-jacoco/values.yaml

```diff
@@ -3,11 +3,11 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: jenkinsxio/ext-jacoco
+  repository: draft
```

Comment:
?

Reply:
yeah looks like you need `jenkinsxio/jx-app-jacoco`

Reply:
Will do.